runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/AbstractWindmillStream.java

```diff
@@ -154,112 +159,169 @@ private static long debugDuration(long nowMs, long startMs) {
 
   /** Reflects that {@link #shutdown()} was explicitly called. */
   protected boolean isShutdown() {
```

Comment:
name the method and variable differently.

Reply:
done